kafka-ui-react-app/src/components/Connect/Details/Actions/Actions.tsx

```diff
@@ -138,7 +138,7 @@ const Actions: React.FC = () => {
           danger
           permission={{
             resource: ResourceType.CONNECT,
-            action: Action.DELETE,
+            action: Action.EDIT,
```

Comment:
why should a `DELETE` operation have and `EDIT` action ?

Reply:
there are 4 actions for KC now. `VIEW`, `EDIT`, `CREATE`, `RESTART`.
i think `DELETE` is some kind of WRITE operation (comepare to READ), so i assigned a `EDIT` action to deleting a KC.
maybe later, `DELETE` will be removed from `EDIT` and be it's own action.
but for now, if it has to be fixed within 4 actions, `EDIT` is appropriate.

Reply:
@Haarolean ?